extension/gulpfile.js

```diff
@@ -88,8 +88,11 @@ gulp.task('browserify', () => {
   ], {read: false})
     .pipe(tap(file => {
       file.contents = browserify(file.path, {
+        fullPaths: true,
```

Comment:
doesn't this really bulk up lighthouse-background.js? Especially with 200+ new files added to it